test/screenshot/spec/mdc-textfield/fixture.scss

```diff
@@ -47,3 +47,11 @@
     transform: translateX(.1px);
   }
 }
+
+.mdc-text-field {
+  width: 100%;
```

Comment:
It is required for text field component and helper line to have same width. So it is very likely that all the existing screenshots break because of this. Or we can apply this to only to screenshots that render helper line.

Reply:
Yes, I intend to document this.